build failure while converting image-augmentation.ipynb (charmap can't decode)

When building on Windows in a Conda environment with `make EVAL=0`:

```
python build/md2ipynb.py guide/module/gluon/image-augmentation.md build/./guide/module/gluon/image-augmentation.ipynb
Traceback (most recent call last):
  File "build/md2ipynb.py", line 30, in <module>
    md2ipynb()
  File "build/md2ipynb.py", line 18, in md2ipynb
    notebook = reader.read(f)
  File "C:\Users\Aaron\AppData\Local\conda\conda\envs\mxnet-docs\lib\site-packages\nbformat\v4\rwbase.py", line 113, in read
    nbs = cast_unicode_py2(fp.read())
  File "C:\Users\Aaron\AppData\Local\conda\conda\envs\mxnet-docs\lib\encodings\cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 2841: character maps to <undefined>
make: *** [Makefile:17: build/./guide/module/gluon/image-augmentation.ipynb] Error 1
```

--- a/python/build/md2ipynb.py
+++ b/python/build/md2ipynb.py
@@ -14,7 +14,7 @@ def md2ipynb():
     do_eval = int(os.environ.get('EVAL', True))
 
     reader = notedown.MarkdownReader(match='strict')
-    with open(input_fn, 'r') as f:
+    with open(input_fn, 'r', encoding="utf8") as f:
         notebook = reader.read(f)
     if do_eval:
         tic = time.time()

